### ISSUE TITLE
Support return types

### ISSUE THREAD

**Issue by GaryJones** [phpro/soap-client#36]
A generated method like:

```php
    /**
     * @return string
     */
    public function getNewUsername()
    {
        return $this->NewUsername;
    }
```

could have a return type, if a config flag was set, so that it generated it as:

```php
   /**
     * @return string
     */
    public function getNewUsername(): string
    {
        return $this->NewUsername;
    }
```

**Comment by veewee** [comment]
Small note: this might be a very tricke feature. Ext-soap does not support XSD simpletypes. If you create a named simpletype with type string, you'll see the name instead of the type string.

**Comment by veewee** [comment]
Fixed in #61
